src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from redminelib import Redmine
+from enum import Enum
+import errno
+import json
+import requests
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+from ..model.feedback import FeedbackModel
+
+
+class FeedbackProject(Enum):
+    dashboard = 46
+
+class FeedbackTracker(Enum):
+    bug = 1
+    feature = 2
+
+class FeedbackSeverity(Enum):
+    minor = '3 - Minor'
+    major = '2 - Major'
+    critical = '1 - Critical'
```

Comment:
I'm thinking of removing severity all together since it is only a part of [bugs](https://tracker.ceph.com/issues/52300.json) and not [features](https://tracker.ceph.com/issues/44851.json). You can check the JSON in the attached links. 
Also it is not working as expected right now. I'm debugging this.

Reply:
Removed severity